node/node.go

```diff
@@ -24,33 +30,39 @@ import (
 type Node struct {
 	app                        monomer.Application
 	genesis                    *genesis.Genesis
-	httpListener               net.Listener
-	wsListener                 net.Listener
+	engineHTTP                 net.Listener
+	engineWS                   net.Listener
+	cometHTTPAndWS             net.Listener
```

Comment:
The goal was to align with how geth and comet behave:

- geth uses separate servers for each transport
- comet uses a single server for everything

But it may not be necessary to do this. I'll look a bit deeper.

Reply:
Will leave it alone for this PR. I made a task to remove the engine http endpoint since the op node only uses websocket.